attribute_set/models/attribute_attribute.py

```diff
@@ -292,6 +292,27 @@ def create(self, vals):
             # before creating the new 'attribute.attribute'
             for key in set(vals).intersection(self.env["ir.model.fields"]._fields):
                 del vals[key]
+            field = self.env["ir.model.fields"].browse(vals["field_id"])
```

Comment:
The create method start to be really loooong, maybe we can take the opportunity to add two new method
- update_create_native_fields(self, vals)
- update_create_custom_fields(self, vals)

and having something like

```
def create(self, vals):
    if vals.get("nature") == "native":
        self._update_create_native_fields(vals)
    else:
       self._update_create_custom_fields(vals)
```


what do you think ?

Reply:
Seems legit, method split :+1: